### PR TITLE
Enable Admins to delete any shared course

### LINKED ISSUE #2259: Admin user can't delete any shared course
## Expected Behaviour
Admin user should be able to delete any course.

## Actual Behaviour
A course that get's shared by deleting a user (select "share all courses") can not be deleted by a admin user.

## Steps to Reproduce
1. delete a user and select "share all courses" from the dropdown
2. try to delete the course as an admin 

## Versions
  - Authoring Tool Version: 0.7.0
  - Framework Version: --

### PR DESCRIPTION
**Pull request by lc-thomasberger**
fixes #2259 

Merges in the Frontend check from https://github.com/adaptlearning/adapt_authoring/pull/1926

### CODE REVIEW

File: frontend/src/modules/projects/views/projectView.js [hunk 1]
```diff
@@ -75,28 +75,16 @@ define(function(require) {
...
+      var textKey = isShared ? 'confirmdeletesharedprojectwarning' : 'confirmdeleteprojectwarning';
 
-      }
       Origin.Notify.confirm({
         type: 'warning',
-        title: Origin.l10n.t('app.deleteproject'),
-        text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.confirmdeleteprojectwarning'),
+        title: Origin.l10n.t('app.' + titleKey),
+        text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.' + textKey),
+        destructive: isShared && !isAdmin, // waive the wait for 'trusted' admin users
```
Comment (canstudios-nicolaw) [inline frontend/src/modules/projects/views/projectView.js]: Hi @lc-thomasberger we discussed this in the stand-up yesterday and we think that we should keep this wait as the visibility of some courses is changing with this release. It's safer to keep this for now.

Comment (lc-thomasberger) [reply frontend/src/modules/projects/views/projectView.js]: Hello, the whole PR?
If we do not merge this PR we must also remove the option "share all courses" for the user delete dropdown.

Comment (canstudios-nicolaw) [reply frontend/src/modules/projects/views/projectView.js]: No not the whole PR. It's fine for an admin to be able to delete any course, but we want the admin to have the 5 second wait before they can delete it

File: frontend/src/modules/projects/views/projectView.js [hunk 2]
```diff
@@ -75,28 +75,16 @@ define(function(require) {
...
+      var textKey = isShared ? 'confirmdeletesharedprojectwarning' : 'confirmdeleteprojectwarning';
 
-      }
       Origin.Notify.confirm({
         type: 'warning',
-        title: Origin.l10n.t('app.deleteproject'),
-        text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.confirmdeleteprojectwarning'),
+        title: Origin.l10n.t('app.' + titleKey),
+        text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.' + textKey),
+        destructive: true,
```
Comment (taylortom) [inline frontend/src/modules/projects/views/projectView.js]: Apologies for the confusion with this, it only needs to apply to shared courses as before
```suggestion
        destructive: isShared,
```

Comment (canstudios-nicolaw) [reply frontend/src/modules/projects/views/projectView.js]: My apologies

**Pull request merged**